src/ask_order.rs

```diff
@@ -73,126 +53,10 @@ pub fn migrate_ask_orders(
     let version_info = get_version_info(store)?;
     let current_version = Version::parse(&version_info.version)?;
 
-    // version support added in 0.15.0, all previous versions migrate to v1 of state data
-    let upgrade_req = VersionReq::parse("<0.15.0")?;
-
-    if upgrade_req.matches(&current_version) {
-        let existing_ask_order_ids: Vec<Vec<u8>> = bucket_read(store, NAMESPACE_ORDER_ASK)
-            .range(None, None, Order::Ascending)
-            .map(|kv_bid: StdResult<Record<AskOrder>>| {
-                let (ask_key, _) = kv_bid.unwrap();
-                ask_key
-            })
-            .collect();
-
-        for existing_ask_order_id in existing_ask_order_ids {
-            let existing_ask_order: AskOrder =
-                bucket_read(store, NAMESPACE_ORDER_ASK).load(&existing_ask_order_id)?;
-            get_ask_storage(store).save(&existing_ask_order_id, &existing_ask_order.into())?
-        }
-    }
+    require_version(">=0.15.0", &current_version)?;
```

Comment:
why is this 0.15.0 and not 0.16.2?

Reply:
See this comment from @brashidnia-figure https://github.com/FigureTechnologies/ats-smart-contract/pull/111#discussion_r1174082377 @brashidnia-figure An explanation would be good, so I could put some comments in the code

Reply:
ya, i know that the actual minimum upgrade on the overall migration is `v0.16.2` (becuse of the bid orders), but for the `migrate_ask_orders()` portion, it would technically be ok for `v0.15.0` since that was the latest version of the `AskOrderV1`